components/review-sections/IdAndResidencyReviewSection.tsx

```diff
@@ -0,0 +1,83 @@
+import { Heading, HeadingLevels, SummaryList } from "lbh-frontend-react";
+import React from "react";
+import PropTypes from "../../helpers/PropTypes";
+import useReviewSectionRows from "../../helpers/useReviewSectionRows";
+import {
+  idAndResidencyProcessSteps,
+  idAndResidencyResidentSteps,
+} from "../../steps/id-and-residency";
+import Storage from "../../storage/Storage";
+
+interface Tenant {
+  fullName: string;
+  id: string;
+  isPresent: boolean;
```

Comment:
```suggestion
  present: boolean;
```

We tend to prefer this style to be consistent with HTML booleans (e.g. `required`, `checked`).